zeebe/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/RecordIndexRouterTest.java

```diff
@@ -142,7 +149,8 @@ void shouldReturnSearchPatternForValueType() {
     final var valueType = ValueType.PROCESS;
 
     // when
-    final var prefix = router.searchPatternForValueType(valueType);
+    final var prefix =
+        router.searchPatternForValueType(valueType, VersionUtil.getVersionLowerCase());
```

Comment:
🔧 I think it would be beneficial to also have a test not using the VersionUtil, to verify that we actual use the parameter not the static